include/gatekeeper_lls.h

```diff
@@ -211,6 +211,16 @@ struct lls_config {
 };
 
 /*
+ * TODO Enable an lcore to subscribe multiple times for the same IP address.
+ * This is needed because it's very hard for the GK blocks avoid duplicates.
+ * Currently, the LLS block has trouble with unsubscribing those duplicates
+ * since the LLS block identifies what to remove using
+ * the pair <IP_address, lcore_id>.
+ * In addition, the LLS block might not accept a subscription because
+ * it saves memory for a single subscription for lcore id.
+ */
```

Comment:
I think we should rewrite the comment to make it a documentation of how LLS works because it is no longer an issue. It was an issue in previous ideas of how GK and GT blocks would use LLS' services. But in the current version of GK blocks, and the idea already discussed for the GT blocks, GK and GT blocks will avoid the duplicates.

For example, GK blocks avoid the duplicates with the help of their hash tables of neighbors that leaves inside of the LPM table. And each GT block will have a hash table of neighbors.